src/a-frame/component/tour.js

```diff
@@ -133,4 +151,13 @@ function dist(p, q) {
   var b = parseFloat(q.y) - parseFloat(p.y)
   var c = parseFloat(q.z) - parseFloat(p.z)
   return Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2) + Math.pow(c, 2))
+}
+
+// Returns true if it is a DOM element
+// https://stackoverflow.com/a/384380/2835973
+function isElement(o){
+  return (
```

Comment:
might be personal style, but I like "early return", which is a technique where you bail from a function if some fundamental assumption is wrong, e.g.:

```javascript
function isElement(o) {
  if (!runtime.isBrowser) return false

  return ( o instanceof HTMLElement && o.nodeType === DOM_ELEMENT )
}
```

I also think that the check for `nodeName` isn't needed or is it?

Reply:
the whole tour component is useless outside of a browser, right?